modules/create_timepoint/php/timepoint.class.inc

```diff
@@ -234,7 +234,9 @@ class Timepoint extends \NDB_Page implements ETagCalculator
             null : $values['psc'];
 
         // validate site entered
-        $site = !empty($values['psc']) ? \CenterID::singleton($values['psc']) : null;
+        $site = !empty($values['psc'])
+        ?  \CenterID::singleton(intval($values['psc']))
+        : null;
```

Comment:
Can you indent these two lines?

Reply:
This indentation came from `npm run lintfix:php`.. I just tried to manually indent it and it looks like PHPCS doesn't complain, so sure.